packages/block-library/src/image/edit.js

```diff
@@ -425,10 +406,25 @@ class ImageEdit extends Component {
 				<Fragment>
 					{ controls }
 					<MediaPlaceholder
-						icon={ <BlockIcon icon={ icon } /> }
-						className={ className }
+						icon={ ! url ? <BlockIcon icon={ icon } /> : <img
+							alt={ __( 'Replace image' ) }
+							title={ __( 'Replace image' ) }
+							width={ '50%' }
```

Comment:
Is setting the width of an `<img>` to `50%` valid? My understanding is that `width` and `height` should reflect the actual width and height of the image. Should we be using CSS for this?

Reply:
MDN says about the width attribute that "For the elements listed here, this establishes the element's width." Historically, before CSS, img width/height was good to have the browser reserve the space b4 the image would load. 

In this particular case though CSS is a better option indeed since the image is already loaded.